[2022-05-19 15:56:44,681] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_validate.dbt_compile scheduled__2022-05-18T15:55:44.156802+00:00 [queued]>
[2022-05-19 15:56:44,714] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: model_level_dbt_great_expectations_dag.dbt_validate.dbt_compile scheduled__2022-05-18T15:55:44.156802+00:00 [queued]>
[2022-05-19 15:56:44,716] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 15:56:44,718] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2022-05-19 15:56:44,719] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 15:56:44,768] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_validate.dbt_compile> on 2022-05-18 15:55:44.156802+00:00
[2022-05-19 15:56:44,802] {standard_task_runner.py:52} INFO - Started process 541 to run task
[2022-05-19 15:56:44,811] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'model_level_dbt_great_expectations_dag', 'dbt_validate.dbt_compile', 'scheduled__2022-05-18T15:55:44.156802+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/model_level_dbt_great_expectations_dag.py', '--cfg-path', '/tmp/tmpyq843dt6', '--error-file', '/tmp/tmpjg5a_t00']
[2022-05-19 15:56:44,813] {standard_task_runner.py:77} INFO - Job 131: Subtask dbt_validate.dbt_compile
[2022-05-19 15:56:44,911] {logging_mixin.py:109} INFO - Running <TaskInstance: model_level_dbt_great_expectations_dag.dbt_validate.dbt_compile scheduled__2022-05-18T15:55:44.156802+00:00 [running]> on host 34ae02bc0b96
[2022-05-19 15:56:45,003] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=luciano.naveiro
AIRFLOW_CTX_DAG_ID=model_level_dbt_great_expectations_dag
AIRFLOW_CTX_TASK_ID=dbt_validate.dbt_compile
AIRFLOW_CTX_EXECUTION_DATE=2022-05-18T15:55:44.156802+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-18T15:55:44.156802+00:00
[2022-05-19 15:56:45,005] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-19 15:56:45,008] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n        dbt compile --profiles-dir /opt/dbt/jaffle_shop --project-dir /opt/dbt/jaffle_shop\n        ']
[2022-05-19 15:56:45,043] {subprocess.py:85} INFO - Output:
[2022-05-19 15:56:51,857] {subprocess.py:89} INFO - 15:56:51  Running with dbt=1.1.0
[2022-05-19 15:56:52,080] {subprocess.py:89} INFO - 15:56:52  Found 5 models, 13 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-19 15:56:52,095] {subprocess.py:89} INFO - 15:56:52
[2022-05-19 15:56:52,389] {subprocess.py:89} INFO - 15:56:52  Concurrency: 1 threads (target='dev')
[2022-05-19 15:56:52,391] {subprocess.py:89} INFO - 15:56:52
[2022-05-19 15:56:53,027] {subprocess.py:89} INFO - 15:56:53  Done.
[2022-05-19 15:56:54,087] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-19 15:56:54,117] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=model_level_dbt_great_expectations_dag, task_id=dbt_validate.dbt_compile, execution_date=20220518T155544, start_date=20220519T155644, end_date=20220519T155654
[2022-05-19 15:56:54,173] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-19 15:56:54,231] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
